### ISSUE TITLE
COCO dataset training has 'noisy' loss function that doesn't drop below 1.5

### ISSUE THREAD

**Issue by martinrkeyser** [AlexeyAB/darknet#4637]
I have downloaded the COCO dataset, used cocotoyolo.jar to extract my 5 classes and started training on my dataset. I followed the instructions to the T (changing all my filters and class numbers in the config file). I left it for 20 000 iterations but the loss function becomes 'noisy' after about 1000 iterations and then just oscillates between 2.5 and 1.5. My dataset consists of about 70 000 images and I used darknet53.conv.74 for my pre-trained weights.

My graph ends up looking like this:
![chart](https://user-images.githubusercontent.com/7796862/71820594-3cb6da00-3098-11ea-856a-b2c1bb6a401c.png)

Has anybody run into this problem? Any help would be greatly appreciated!

**Comment by AlexeyAB** [comment]
This is normal for MS COCO

**Comment by martinrkeyser** [comment]
Good morning. Thank you for your seedy response.

This might be a stupid question, but what is 'MS' COCO?

Then, is there a dataset that you can recommend? I trained your version of YOLO on the 2017 dataset for coco for 20 000 iterations and after running the detector on a few test images, I did not get great results, I get very low confidence on my detections:

![predictions](https://user-images.githubusercontent.com/7796862/71872511-55210600-3125-11ea-896e-1810fa4efe1c.jpg)
motorcycle: 47%
person: 41%
person: 55%
person: 40%
person: 35%
person: 35%
person: 30%

Lastly, I read somewhere that when you are training, you should adjust your subdivisions and batch to fit your graphics capabilities but when detecting you should set them to 1 and 1. Is this correct?

**Comment by AlexeyAB** [comment]
MS - Microsoft
confidence - doesn't mean anything. Only accuracy (mAP) is important to you.
you should adjust only your subdivisions to fit your graphics capabilities
